task-2/src/components/Movie/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './index.css';
+
+function Movie(moviesInfo) { 
```

Comment:
Consider keeping the name "props" instead of "moviesInfo".

Reply:
The correct naming convention of props has been restored.